public/css/data-discovery-datasets.css

```diff
@@ -7,14 +7,6 @@ table.datasets td {
     padding: 5px 10px;
 }
 
-table.datasets tr:hover td {
```

Comment:
In this case, I think the CSS should stay here, just change the color name to something like: `--dataset-table-hover-color' and set that in each theme template.

Reply:
Actually, never mind my second part of the comment, you did that.